config

```diff
@@ -0,0 +1,15 @@
+config = {
```

Comment:
Not everyone will be needing to use the postgres adapter. Can you propose a better place to store this config file ? Maybe in the adapter itself ?